fftoml/fftoml_test.go

```diff
@@ -9,9 +9,12 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/peterbourgon/ff/v2"
+	"github.com/peterbourgon/ff/v2/fftoml"
```

Comment:
I'm not happy with changing the import path. Is there no way around this?

Reply:
This is sufficiently unacceptable to me as an outcome that I am inclined to break compatibility promises and stay on v0/v1 in perpetuity. Package identifier and version are and ought to be independent variables, modules' insistence on co-mingling them is an unacceptable design choice.

Reply:
(While I personally dislike the versioned import path for a number of reasons, I would see breaking SemVer as a signal that I can't rely on a module to be easy to keep up to date, and I would be discouraged from using it. A standard way to send that signal is to stay on v0.)